collect_app/src/main/java/org/odk/collect/android/http/CollectServerClient.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (C) 2011 University of Washington
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.odk.collect.android.http;
+
+import android.net.Uri;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import org.kxml2.io.KXmlParser;
+import org.kxml2.kdom.Document;
+import org.odk.collect.android.utilities.DocumentFetchResult;
+import org.odk.collect.android.utilities.WebCredentialsUtils;
+import org.xmlpull.v1.XmlPullParser;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.HashMap;
+
+import javax.inject.Inject;
+
+import timber.log.Timber;
+
+/**
+ * Common utility methods for managing the credentials associated with the
```

Comment:
Do you think we could modify this comment to make it a bit more meaningful for since it's meaning has changed since using this client class instead of an utils?

Reply:
Updated comments in this class and removed redundant ones.